keyvalue/client/src/main/java/alluxio/hadoop/mapreduce/KeyValueOutputCommitter.java

```diff
@@ -109,6 +109,6 @@ public void abortTask(TaskAttemptContext taskContext) {
    * @return the temp directory name
    */
   public static String getPendingDirName() {
-    return FileOutputCommitter.TEMP_DIR_NAME;
+    return FileOutputCommitter.PENDING_DIR_NAME;
```

Comment:
Unfortunately we're stuck with the deprecated constant as long as we support hadoop 1: https://hadoop.apache.org/docs/r1.2.1/api/org/apache/hadoop/mapreduce/lib/output/FileOutputCommitter.html